Dockerfile

```diff
@@ -8,19 +8,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG BUILDER=golang:1.19-alpine
+ARG BUILDER=public.ecr.aws/eks-distro-build-tooling/golang:1.19-gcc
 ARG BASE_IMAGE=public.ecr.aws/eks-distro/kubernetes/go-runner:v0.13.0-eks-1-23-9
 
-FROM ${BUILDER} AS build
+FROM --platform=$BUILDPLATFORM ${BUILDER} AS build
```

Comment:
Does this require `docker buildx build` in the [Makefile](https://github.com/kubernetes-sigs/aws-encryption-provider/blob/master/Makefile#L19)?

Reply:
Is this necessary?

Reply:
Yes, `BUILDPLATFORM` only gets populated when using `docker buildx`.